### ISSUE TITLE
iOS title bar overlaps

### ISSUE THREAD

**Issue by stillatmylinux** [sunnycupertino/cordova-plugin-admob-simple#3]
Would it be possible to reduce the webview by about 20px on top for iOS so the status bar doesn't overlap when there is a banner ad at the bottom?

**Comment by sunnycupertino** [comment]
When the ad is not displayed does the overlap still happen? It's not really within the bounds of the plugin to change the size of the webview. It either ads the ad to the webview, or ads it over the top, using the 'overlap' variable which you can set in the javascript.

If the overlap is occurring because of the ad (eg it doesn't occur without it) then set overlap: true in the plugin settings and the webview should  stay the same size.

Let me know how you go.

**Comment by stillatmylinux** [comment]
I figured out my problem. I was assigning ca-app-pub-xxxxxxxxxxx/xxxxxxxxxxx using a local variable.  When I left out the iOS interstitial ad, my code ended up assigning false to that option.  Like this:

```
admobid.interstitial = false; // Error.  I should have this as an empty string instead of false
window.plugins.AdMob.setOptions({
    publisherId: admobid.banner,
    interstitialAdId: admobid.interstitial,
```

When I did this my app would sometimes crash or have issues with the statusbar overlapping.  Also it would ignore the overlap, offsetTopBar and isTesting options.  Once I changed that to an empty string everything worked.
